app/assets/stylesheets/pages/_home.scss

```diff
@@ -1,6 +1,5 @@
 // Specific CSS for your home-page
 .page-container {
-  background-image: url("doctor-light.png");
```

Comment:
Thats not quite enough.
The other code in .page-container is now of no use and we should avoid dead code :)